### ISSUE TITLE
Add address to project page

### ISSUE THREAD

**Issue by AlexZhidkov** [AlexZhidkov/PracLearn#7]
"As a Student selecting a project to apply for I want to know the address of the organisation so I know how far I would need to commute to work there"

Show address on the project page above Apply button.
Add a button to redirect to the on Google maps.

**Comment by crab-coding** [comment]
Hi Alex.
I'd like to work on this project. Can we discuss the opportunity?

**Comment by AlexZhidkov** [comment]
Hi, yes, sure, you're welcome. Fork the repo, make this change and create Pull Request. If you have any questions, please ask.

**Comment by crab-coding** [comment]
Yeah but I'd like to make a contract for this position. Can we talk about that?

**Comment by AlexZhidkov** [comment]
Sure. what can you do for us?

**Comment by crab-coding** [comment]
Feel free to check my contributions and public repos on my github account.
I am very good at Javascript frameworks like React.
If you are still interested in we can we talk on Skype. Nathanmwork@outlook.com
